### ISSUE TITLE
TEG broke in such a way that it didn't even need gases to make power.

### ISSUE THREAD

**Issue by Eneocho** [d3athrow/vgstation13#31588]
#### Description of issue
We had made a real fancy R-UST, 8x8 with double layered heat pipes and all the good stuff. But when time came for the TEG to make power, it froze at exactly 12.8447MW. We first thought it was a flow issue, but when disconnecting the circulators it still made that exact amount of power.
I had made this design before and I got around 50GW, give or take a few from different plasma amounts.

Pic related:
![broken reality two TEG BOOGALOO](https://user-images.githubusercontent.com/67024428/145670489-fc0f374b-5f73-4fd4-8081-b082c3fb207e.png)

That generator piece made power without circulators. And when swapping from that generator to another, the second one made exactly 0W, however, when returning to the original generator the power went back to 12.8447MW. We assumed the power system froze or crashed, since kernikov tried to restart the power controller but nothing happened.
#### Difference between expected and actual behavior
We expected niggawatts.
Got hauntspace magic.
#### Steps to reproduce
Snaxi-
Make a RUST like the one in the pic.
Connect to TEG like in the pic.
???
HAUNTSPACE.

[bugfix][snowmap]

**Comment by SonixApache** [comment]
![image](https://user-images.githubusercontent.com/8468269/145679807-0b4f04ce-ce3b-4694-867b-04dd995d8c4a.png)
this is probably important to include and also the actual cause

**Comment by Eneocho** [comment]
Yes, forgot to add that, but yes. At one point the cold circulator inlet was also hot for some reason.
